blocks/header/header.css

```diff
@@ -273,3 +312,59 @@ header nav .nav-sections ul > li > ul > li {
 header nav .nav-tools {
   grid-area: tools;
 }
+
+.languagenavigation {
+  width: 10%;
+}
+
+.cmp-languagenavigation__item--active {
+  cursor: pointer;
+  display: -webkit-inline-box;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+nav.cmp-languagenavigation {
+  position: relative;
+}
+
+nav.cmp-languagenavigation [lang=EN-US]:before {
+  content: url(../../../etc.clientlibs/24petwatch/clientlibs/clientlib-site-base/resources/images/USAFlag.svg);
```

Comment:
please move SVGs to either github (https://www.hlx.live/developer/block-collection/icons) or inline them in CSS. But do not reference them from AEM. The relative URL won't resolve since we dont have `/etc.clientlibs/...` in Franklin